app/assets/src/components/views/SampleViewV2/ReportFilters.jsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { flatten, forEach, getOr, transform, values } from "lodash/fp";
+
+import BackgroundModelFilter from "~/components/views/report/filters/BackgroundModelFilter";
+import CategoryFilter from "~/components/views/report/filters/CategoryFilter";
+import MetricPicker from "~/components/views/report/filters/MetricPicker";
+import MinContigSizeFilter from "~/components/views/report/filters/MinContigSizeFilter";
+import NameTypeFilter from "~/components/views/report/filters/NameTypeFilter";
+import SearchBox from "~ui/controls/SearchBox";
+import SpecificityFilter from "~/components/views/report/filters/SpecificityFilter";
+import ThresholdFilterDropdown from "~ui/controls/dropdowns/ThresholdFilterDropdown";
+import { logAnalyticsEvent } from "~/api/analytics";
+
+import { CATEGORIES, THRESHOLDS, TREE_METRICS } from "./constants";
+import cs from "./report_filters.scss";
+
+class ReportFilters extends React.Component {
+  handleFilterChange = ({ key, value }) => {
+    const { onFilterChanged } = this.props;
+
+    logAnalyticsEvent("SampleView_filter_changed", {
+      key,
+      value,
+    });
+    onFilterChanged({ key, value });
+  };
+
+  render = () => {
+    const { backgrounds, sampleId, selected, view } = this.props;
+
+    return (
+      <div className={cs.filterList}>
+        <div className={cs.filterListElement}>
+          <SearchBox
```

Comment:
Searching doesn't seem to work locally for me (even on the old version of the report page). Made a note to self to test on staging!

Reply:
Do you have elastic search running?